internal/rpcserver/rpcwebsocket.go

```diff
@@ -124,8 +125,7 @@ func (s *Server) WebsocketHandler(conn *websocket.Conn, remoteAddr string, authe
 		return
 	}
 	s.ntfnMgr.AddClient(client)
-	client.Start()
-	client.WaitForShutdown()
+	client.Run(context.TODO())
```

Comment:
A todo here to pass the dcrd parent context to synchronize lifecycles would be good.

Reply:
Disregard, just noticed this is being done in #3026.